packages/runtime/yaml-config.graphql

```diff
@@ -16,6 +16,10 @@ type Query {
   Backend cache
   """
   cache: Cache
+  """
+  Merge method
+  """
+  merger: String
```

Comment:
I wonder if we'll need additional configuration here in the future. Something like: 

```
merger:
  type: federation
  config: ...
```

So maybe it's better to use an object here?

Reply:
There is no configuration for them so that would make it longer which is unnecessary.